app/controllers/application_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  def after_sign_in_path_for(_resource)
```

Comment:
- [ ] Upon signning up, I am getting error. Please fix this. 

![image](https://user-images.githubusercontent.com/58504655/228990858-e5650727-746f-49b7-83ae-766497964b06.png)